.gitignore

```diff
@@ -1,11 +1,5 @@
-.DS_Store
 *.pyc
-settings.py
-test.db
-*.wpr
-.project
-.pydevproject
-.settings
+development.py
 .sass-cache
-geekup/static/img/uploads
 error.log
+testing.py
```

Comment:
`settings.py`, `development.py` and `testing.py` should be prefixed with `instance/` so that other files that share the name aren't accidentally excluded.